clients/rust/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mpl-bubblegum"
-version = "1.4.0"
+version = "1.5.0"
```

Comment:
Note: you don't need to update this version as cargo release does this as part of our client publish CI.